apps/full-site-editing/full-site-editing-plugin/homepage-posts/index.php

```diff
@@ -0,0 +1,48 @@
+<?php
+define( 'NEWSPACK_BLOCKS__BLOCKS_DIRECTORY', 'dist/' );
+define( 'NEWSPACK_BLOCKS__PLUGIN_DIR', plugin_dir_path( __FILE__ ) );
+define( 'NEWSPACK_BLOCKS__VERSION', '1.0.0-alpha.17' );
+
+function homepage_posts_block_name( $name ) {
+	if ( 'newspack-blocks/homepage-articles' === $name ) {
+		return 'a8c/homepage-posts';
+	}
+	return $name;
+}
```

Comment:
why not define the name when the block is registered instead of using a hook?

Reply:
The actual registration of the block is done in synchronized code and thus the natural way to update it there is using a filter. We cannot touch that code directly